parts/k8s/kubernetesinstalls.sh

```diff
@@ -16,6 +16,10 @@ removeMoby() {
     sudo apt-get purge -y moby-engine moby-cli
 }
 
+removeDockerEngine() {
+	apt_get_remove 20 30 120 docker-engine
```

Comment:
Should `removeMoby` and `removeDockerEngine` be using the same method for uninstall?

indent looks off

Reply:
Possibly. The reason I did not is `purge` was removing `/etc/docker/daemon.json`, I'm not sure why it wasn't replaced later on... but in any case it caused the deployment to fail, timing out on `[ -f /etc/docker/daemon.json ]` to return ok.